### PR TITLE
GFORMS-2502 - Upgrade gform-frontend repository to play 3.

### PR DESCRIPTION
**Pull request by DursunGundogan**
Upgrade scala version to 2.13.12

### CODE REVIEW

File: test/uk/gov/hmrc/gform/binders/ValueClassBinderSpec.scala [hunk 1]
```diff
@@ -1,57 +0,0 @@
...
- */
-
-package uk.gov.hmrc.gform.binders
-
-import org.scalatestplus.play.PlaySpec
-import uk.gov.hmrc.gform.models.FastForward
-import uk.gov.hmrc.gform.sharedmodel.formtemplate.SectionOrSummary
-import uk.gov.hmrc.gform.sharedmodel.formtemplate.SectionNumber
-
-class ValueClassBinderSpec extends PlaySpec {
```
Comment (josef-vlach) [inline test/uk/gov/hmrc/gform/binders/ValueClassBinderSpec.scala:24]: Why is this deleted?

Comment (DursunGundogan) [reply test/uk/gov/hmrc/gform/binders/ValueClassBinderSpec.scala:24]: It's testing [`ValueClassBinder.fastForwardListBinder`](https://github.com/hmrc/gform-frontend/pull/2077/files#diff-20adfff1866bae740f327735cd1c114c81a1220eaa1dd882c907a326a47cb827L244). I think we don't need this implementation.

**Review by josef-vlach — APPROVED** [review]
